R/compute_duration.R

```diff
@@ -37,7 +37,7 @@
 #'   Default: 'days'
 #'
 #'   Permitted Values: 'years', 'months', 'weeks', 'days', 'hours', 'minutes',
-#'   'min', 'seconds', 'sec'
+#'   'min', 'seconds', 'sec', capitalized versions of these terms is accepted too
```

Comment:
This should be updated as we accept even more values now. (`in_unit` and `age_unit` should be updated as well)